5calls/app/src/main/java/org/a5calls/android/a5calls/model/Contact.java

```diff
@@ -13,6 +13,9 @@
  *
  */
 public class Contact implements Parcelable {
+    public static final String AREA_HOUSE = "US House";
+    public static final String AREA_SENATE = "US Senate";
+
```

Comment:
if you want to add the other areas we support as constants here for completeness, they are:
`StateLower`,`StateUpper`,`Governor`,`AttorneyGeneral`,`SecretaryOfState`